src/App.test.js

```diff
@@ -1,9 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import { IntlProvider } from 'react-intl';
+import messages_es from './i18n/es.json';
+import messages_en from './i18n/en.json';
+
+const messages = {
+  es: messages_es,
+  en: messages_en,
```

Comment:
Why are you importing all languages if then there is a constant to use only `en`?

Reply:
Yes, you're right, in the test we can keep the one that we use.